### ISSUE TITLE
Log kann nicht gespeichert werden

### ISSUE THREAD

**Issue by CarG70** [evcc-io/app#19]
Wenn ich einen Log herunterladen möchte, erscheint lediglich eine leere Seite im Browser mit dem Hinweis "Unauthorized" oben links. Über die WebApp funktioniert es.

**Comment by naltatis** [comment]
Ja, liegt daran, dass wir für den Download aktuell einfach in den Systembrowser verlinken (`Linking`). Der hat natürlich die erforderlichen Berechtigungen nicht. Ich bin mir noch nicht ganz sicher, was eine geschickte Alternative ist direkt einen Download/Share-Sheet aus der Webview zu veranlassen.

**Comment by Maschga** [comment]
Ist das immer noch ein Bug? Zumindest auf Android und dem Demo Server von Evcc kann ich diesen Fehler nicht reproduzieren:
 https://github.com/user-attachments/assets/24dd01c8-c33d-4ec8-a526-306081e982f7

**Comment by naltatis** [comment]
Ja, auf iOS hab ich hier noch keine schöne Lösung.

**Comment by Maschga** [comment]
Dann ist das nur ein Problem von iOS?

**Comment by naltatis** [comment]
Wenn du sagst, dass es auf Android geht (hatte ich nicht getestet) ist das vmtl. richtig. Ich hab den Titel mal entsprechend angepasst.

**Comment by Maschga** [comment]
Ja, das ist super. Da wollte ich auch hinaus. :D Vielleicht kann man auch dafür Labels erstellen.

**Comment by naltatis** [comment]
Gute Idee, ich hab jetzt ein `iOS` und `Android` Label erstellt. Das wäre dann für Issues, die nur für eine der beiden Plattform relevant sind.
